.gitignore

```diff
@@ -4,4 +4,7 @@
 *.so
 
 # example binary
-examples/examples
\ No newline at end of file
+examples/examples
+
+# idea
+.idea/
```

Comment:
Please make the `gitignore` change a separate commit, and make sure the last line is terminated with a `\n`.